packages/block-library/src/site-title/edit/index.js

```diff
@@ -15,10 +15,13 @@ import {
 	InspectorControls,
 	BlockControls,
 	useBlockProps,
+	__experimentalUseFontAppearanceStyles,
```

Comment:
```suggestion
	__experimentalUseFontAppearanceStyles as useFontAppearanceStyles,
```

I think the pattern usually adopted in Gutenberg is to import the experimental API under what its stabilized name would be to reduce future code changes and possibly prevent some missed updates.

Reply:
Done!, thanks for the suggestion.